src/main/java/seedu/application/model/application/Application.java

```diff
@@ -80,6 +98,20 @@ public boolean isSameApplication(Application otherApplication) {
                 && otherApplication.getPosition().equals(getPosition());
     }
 
+    /**
+     * Returns a new Application object with its archive status set to true and other attributes remain the same.
+     */
+    public Application setToArchive() {
+        return new Application(company, contact, email, position, date, tags, true);
+    }
+
+    /**
+     * Returns a new Application object with its archive status set to false and other attributes remain the same.
+     */
+    public Application retrieveFromArchive() {
```

Comment:
I think the method will be clearer if it is changed to `setToUnarchive()`
```suggestion
    public Application setToUnarchive() {
```

Reply:
For this I was just thinking from the aspect on using it in retrieve command. It will be more consistent on naming it as retrieve I guess?

Reply:
Okay, ya, small issue actually. I am good with adhering to the consistency too. 👍